core/src/main/java/org/elasticsearch/search/suggest/SuggestionBuilder.java

```diff
@@ -197,7 +197,18 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
 
     protected abstract SuggestionBuilder<T> innerFromXContent(QueryParseContext parseContext, String name) throws IOException;
 
-    protected abstract SuggestionContext build(QueryShardContext context) throws IOException;
+    public SuggestionContext build(QueryShardContext context) throws IOException {
+        SuggestionContext suggestionContext = innerBuild(context);
+        // copy over common settings to each suggestion builder
+        populateCommonFields(context.getMapperService(), suggestionContext);
```

Comment:
+1 on calling it inside innerBuild, so we can have it in the same order like in thr current parse element. It can still stay a common method in SuggestionBuilder though, no?